cassandra-analytics-release/prepare_release.sh

```diff
@@ -0,0 +1,349 @@
+#!/bin/bash
+
+#set -o xtrace
+
+##### BEFORE YOU BEGIN #####
+
+# 1. Ensure your gpg configuration in ~/.gradle/gradle.properties
+#    For example:
+#    signing.gnupg.keyName=<your-key>
+#    signing.gnupg.passphrase=<your-passphrase>
+#    #signing.gnupg.executable=gpg # optional
+#    #signing.gnupg.useLegacyGpg=true # optional
+#    #signing.gnupg.homeDir=gnupg-home # optional
+#    #signing.gnupg.optionsFile=gnupg-home/gpg.conf # optional
+
+# 2. Ensure your maven credentials are configured in ~/.gradle/gradle.properties
+#    For example:
+#    maven.repository.url=https://repository.apache.org/service/local/staging/deploy/maven2
+#    maven.username=<asf-username>
+#    maven.password=<asf-password>
+
+##### TO EDIT #####
+
+asf_username="${asf_username:-$USER}"
+asf_username="bernardobotella"
+
+if [ "x${asf_username}" != "x${USER}" ] ; then
+  echo "Using ASF username ${asf_username}"
+fi
+echo "Using ASF username ${asf_username}"
+# The name of remote for the asf remote in your git repo
+git_asf_remote="${git_asf_remote:-origin}"
+
+if [ "x${git_asf_remote}" != "xorigin" ] ; then
+  echo "Using git ASF remote ${git_asf_remote}"
+fi
+
+# Where you want to put the mail draft that this script generate
+mail_dir="$HOME/Mail"
+
+###################
+# prerequisites
+
+command -v svn >/dev/null 2>&1 || { echo >&2 "subversion needs to be installed"; exit 1; }
+command -v git >/dev/null 2>&1 || { echo >&2 "git needs to be installed"; exit 1; }
+command -v shasum >/dev/null 2>&1 || { echo >&2 "shasum needs to be installed"; exit 1; }
+
+###################
+#asf_git_repo="${asf_git_repo:-https://gitbox.apache.org/repos/asf}"
+asf_git_repo="${asf_git_repo:-https://github.com/bbotella}"
```

Comment:
Ideally this should not be changed, and you should be able to run the script with the `asf_git_repo` set to `https://github.com/bbotella` as part of your command variables

```suggestion
asf_git_repo="${asf_git_repo:-https://gitbox.apache.org/repos/asf}"
```